event-store-mem/README.md

```diff
@@ -0,0 +1,55 @@
+event-store-mem: Event Store that stores events in memory
+=========================================================
+
+Level: Beginner  
+Technologies: JavaEE  
+Summary: Example of Event Store provider
+Target Product: Keycloak  
+Source: <https://github.com/keycloak/keycloak-quickstarts>
+
+What is it?
+-----------
+
+This example shows how to implement and deploy an event store provider that stores events in memory.
+You can extend the server's functionality in this way by implementing your own provider. For more information see our server developer guide.
+
+After the provider is deployed and enabled in Keycloak, log out and log in back. To enable the event provider manually go to [Events Config](http://localhost:8180/auth/admin/master/console/#/realms/master/events-settings) tab in the admin console and enable login events and admin events by a toggle buttons. You can configure what event types should be stored. Save the changes afterwards. Go to [Login Events](http://localhost:8180/auth/admin/master/console/#/realms/master/events) tab in the admin console where you can see login events showed from the Event store provider.
+Similarly in [Admin Events](http://localhost:8180/auth/admin/master/console/#/realms/master/admin-events) tab you can see generated admin events. To generate an admin event you can for example create an user.
+
+
```

Comment:
This still needs to detail how to enable the event provider in Keycloak manually. Currently configuration updates are missing. https://github.com/keycloak/keycloak/blob/master/examples/providers/event-store-mem/README.md